includes/Core/Assets/Assets.php

```diff
@@ -274,6 +274,7 @@ function ( $tag, $handle ) use ( $assets ) {
 	 */
 	private function enqueue_minimal_admin_script() {
 		$this->enqueue_asset( 'googlesitekit_admin' );
+		$this->enqueue_asset( 'googlesitekit-api' );
```

Comment:
I'm not sure if this should be enqueued yet. Was this maybe just for testing?

Reply:
Ah, shoot, yes! My bad, though… upon further reflection since this is all being applied against a branch anyway, I wonder why not to do this?

I think it's best to leave it in—if anything we could hide it behind a flag? I've added that for now, but let me know what you think.

Reply:
@felixarntz thoughts?

Reply:
For now I've added a117dc8f852c839d78742f71d0a9edea0bee5cff—I don't think we have a more robust feature-flag setup (though if we do I'm happy to use it instead), so at least we keep the JSR code away in a flag. Should make it easier to test against too.

Reply:
I think we should remove the enqueuing logic for now, even based on a constant as it is now. Let's only add logic like this when it becomes relevant.